condor_watch_q.py

```diff
@@ -635,15 +637,24 @@ def row_data_from_job_state(clusters):
                 active_job_ids.append("{}.{}".format(cluster.cluster_id, proc_id))
 
     row_data[TOTAL] = sum(row_data.values())
+    active_job_ids.sort(key = sortByCluster)
 
-    if len(active_job_ids) > 3:
+    if len(active_job_ids) == 0:
+        row_data[MIN_JOB_ID] = "0"
+    else:
+        row_data[MIN_JOB_ID] = active_job_ids[0]
+
+    if len(active_job_ids) > 2:
         active_job_ids = [active_job_ids[0], active_job_ids[-1]]
         row_data[ACTIVE_JOBS] = " ... ".join(active_job_ids)
     else:
         row_data[ACTIVE_JOBS] = ", ".join(active_job_ids)
 
     return row_data
 
+def sortByCluster(cluster):
```

Comment:
This function is unnecessary; it could be an inlined lambda at line 640: `key = lambda jobid: jobid.split(".")` (then we sort on the proc id as well, since there's no reason not to).

Incidentally, make sure you're following pep8 naming conventions.